apps/avatax/src/pages/api/webhooks/order-cancelled.ts

```diff
@@ -139,12 +141,22 @@ export default wrapWithLoggerContext(
               .json({ message: `App is not configured properly for order: ${payload.order?.id}` });
           }
 
-          await taxProvider.cancelOrder(
-            {
-              avataxId: cancelledOrderInstance.getAvataxId(),
-            },
-            providerConfig.value.avataxConfig.config,
-          );
+          try {
+            await taxProvider.cancelOrder(
+              {
+                avataxId: cancelledOrderInstance.getAvataxId(),
+              },
+              providerConfig.value.avataxConfig.config,
+            );
+          } catch (e) {
+            if (e instanceof AvataxOrderCancelledAdapter.DocumentNotFoundError) {
+              logger.warn("Document was not found in AvaTax. Responding 400", {
+                error: e,
+              });
+
+              return res.status(400).end();
+            }
```

Comment:
**Question:** what about other type of error? Shoudn't we have here 500 returned and error logged to Sentry?

Reply:
yea, good point